### PR TITLE
bugfix: remove jiayouba233's third_party binary resource dependency

### PR DESCRIPTION
**Pull request by Certseeds**
- [x] Bug fix
- [x] Documentation update

<body>

1. do not use absolute path,use relative path
2. use third_party repo's binary resource is dangerous

<footer>

### CODE REVIEW

**Comment by Certseeds** [comment]
fix #145

File: 外交部发言精选/赵立坚/README.md [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
- ![FQRYOs0XMAMU0UZ](https://user-images.githubusercontent.com/72449367/163700430-f8375c4c-3bb6-4059-8efd-ea087fd085c7.jpg)
 
+![偷着乐](./偷着乐.jpg)
+
+![FQRYOs0XMAMU0UZ](./背后黑白偷着乐.jpg)
 ## 赵立坚用叙利亚照片指美国在阿富汗“一炸一路”，遭原作者打脸 [原文链接](https://chinadigitaltimes.net/chinese/676267.html)
 
-![image-1643364942073](https://user-images.githubusercontent.com/72449367/163700740-7cc5916b-e3bf-4b0d-a139-7e306fe459ca.png)
-[![image-1643395155577](https://user-images.githubusercontent.com/72449367/163700768-8ab8a6d2-c38e-4609-acc2-e3a3a708b436.png)](https://twitter.com/AliHajSuleiman/status/1486751480954732548)
+![image-1643364942073](./image-1643364942073.png)
```
Comment (the-run-philosophy) [inline 外交部发言精选/赵立坚/README.md:15]: maybe rename?

File: 外交部发言精选/赵立坚/README.md [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
+
+[![image-1643395155577](./image-1643395155577.png)](https://twitter.com/AliHajSuleiman/status/1486751480954732548)
```
Comment (the-run-philosophy) [inline 外交部发言精选/赵立坚/README.md:17]: this too

File: 新冠疫情相关/外交部发言/赵立坚.md [hunk 3]
```diff
@@ -1,2 +1,3 @@
 ![FQRnSieX0AQ4LxG](https://user-images.githubusercontent.com/72449367/163669608-1d091ebe-82c8-4a93-ad6c-d3af9f4a4436.jpg)
-![FQRYOs0XMAMU0UZ](https://user-images.githubusercontent.com/72449367/163669609-97c7fae1-0fad-4e13-a82e-6ecbc45a1eeb.jpg)
+
+![FQRYOs0XMAMU0UZ](https://raw.githubusercontent.com/The-Run-Philosophy-Organization/run/main/外交部发言精选/赵立坚/背后黑白偷着乐.jpg)
```
Comment (the-run-philosophy) [inline 新冠疫情相关/外交部发言/赵立坚.md:3]: (背后黑白偷着乐.jpg)?

**Review by the-run-philosophy — CHANGES_REQUESTED** [review]

File: 外交部发言精选/赵立坚/README.md (hunk 1 above)
Comment (Certseeds) [reply 外交部发言精选/赵立坚/README.md]: done

File: 外交部发言精选/赵立坚/README.md (hunk 2 above)
Comment (Certseeds) [reply 外交部发言精选/赵立坚/README.md]: done

**Review by the-run-philosophy — APPROVED** [review]
thanks

**Pull request merged**